load_generation_candy_machine/src/initialize.rs

```diff
@@ -0,0 +1,285 @@
+        is_mutable: true,
+        retain_authority: true,
+        go_live_date: Some(0),
+        end_settings: None,
+        hidden_settings: None,
+        whitelist_mint_settings: None,
+        gatekeeper: None,
+    };
+
+    initialize_candy_machine(
+        &candy_machine,
+        &payer,
+        &authority.pubkey(),
+        candy_data,
+        solana_client.clone(),
+    )
+    .await?;
+
+    // add_all_config_lines(&candy_machine.pubkey(), &authority, solana_client.clone()).await?;
+    //  candy_manager.set_collection(context).await.unwrap();
+
+    Ok(candy_machine.pubkey())
+}
+
+pub async fn make_a_candy_machine_v3(
+    solana_client: Arc<RpcClient>,
+    payer: Arc<Keypair>,
+) -> Result<Pubkey, ClientError> {
+    let (candy_machine, authority, minter) =
+        init_candy_machine_info(payer.clone(), solana_client.clone()).await?;
+
+    solana_client
+        .clone()
+        .request_airdrop(&payer.clone().pubkey(), LAMPORTS_PER_SOL * 100)
+        .await?;
+
+    let candy_data = CandyMachineDataV3 {
+        items_available: 3,
+        symbol: DEFAULT_SYMBOL.to_string(),
+        seller_fee_basis_points: 500,
+        max_supply: 0,
+        is_mutable: true,
+        creators: vec![CandyMachineCreatorV3 {
+            address: authority.pubkey(),
+            verified: true,
+            percentage_share: 100,
+        }],
+        config_line_settings: None,
+        hidden_settings: None,
+    };
+
+    initialize_candy_machine_v3(
+        &candy_machine,
+        payer.clone(),
+        &authority.pubkey(),
+        candy_data,
+        solana_client.clone(),
+    )
+    .await?;
+
+    // add_all_config_lines(&candy_machine.pubkey(), &authority, solana_client.clone()).await?;
+    //  candy_manager.set_collection(context).await.unwrap();
+
+    Ok(candy_machine.pubkey())
+}
+
+pub fn make_config_lines(start_index: u32, total: u8) -> Vec<ConfigLine> {
+    let mut config_lines = Vec::with_capacity(total as usize);
+    for i in 0..total {
+        config_lines.push(ConfigLine {
+            name: format!("Item #{}", i as u32 + start_index),
+            uri: format!("Item #{} URI", i as u32 + start_index),
+        })
+    }
+    config_lines
+}
+
+pub async fn add_all_config_lines(
+    candy_machine: &Pubkey,
+    authority: &Keypair,
+    solana_client: Arc<RpcClient>,
+) -> Result<(), ClientError> {
+    solana_client
+        .clone()
+        .request_airdrop(&authority.pubkey(), LAMPORTS_PER_SOL * 100)
+        .await?;
+
+    let candy_machine_account = solana_client.get_account(candy_machine).await?;
+
+    println!("candy {:?}", candy_machine_account);
+
+    let candy_machine_data =
+        CandyMachine::try_deserialize(&mut candy_machine_account.data.as_ref()).unwrap();
+
+    let total_items = candy_machine_data.data.items_available;
+    for i in 0..total_items / 10 {
+        let index = (i * 10) as u32;
+        let config_lines = make_config_lines(index, 10);
+        // add_config_lines(
```

Comment:
All these kind of commented out things, its not clear if they are going to be fixed now or not.  I think if we want to complete this PR without fixing everything, any commented out code like this should be clearly marked with a TODO and then list a Linear task of what the work is.